_posts/2018-10-25-released-v5.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Released Chainer/CuPy v5.0.0
+layout: post
+categories: Announcement
+author: Seiya Tokui
+---
+
+We have released Chainer and CuPy v5.0.0 today!
+This is a major release that introduces several new features.
+
+The following is a list of selected updates. Full updates can be found in the release notes: [Chainer](https://github.com/chainer/chainer/releases/tags/v5.0.0), [CuPy](https://github.com/cupy/cupy/releases/tag/v5.0.0).
+
+- **Static subgraph optimization** *(experimental)*.
+  By applying `@static_graph` decorator to the static part of your computation (which uses the same graph at every iteration), the computational graph of that part is cached and reused.
+  For fully-static models, we are observing 20-60% speed up in most cases.
+  Example code modified for the static subgraph feature can be found [here](https://github.com/chainer/chainer/tree/v5/examples/static_graph_optimizations).
+- **Float16 support**.
+  Using half-precision float is made much easier!
+  Since recent GPU technologies are focusing on half and mixed precision computations, using float16 is crucial for fully utilizing the latest hardware performance.
+  In Chainer v5, the default floating point dtype is configurable via `CHAINER_DTYPE` environment variable or `config.dtype` entry.
+  Using this feature, you can start trying float16 without modifying your code in most cases.
+  Many classes and functions are also fixed to support float16 inputs and parameters.
+- **ChainerMN integration**.
+  [ChainerMN](https://github.com/chainer/chainermn) is an add-on package of Chainer for distributed deep learning.
```

Comment:
I think it'd better be a past tense like: ChainerMN was an add-on package ..., but is no longer an add-on and has become officially built-in module o Chainer v5.

Reply:
Looks good. Applied.